charts/kubebb-core/templates/tekton-resource-configmap.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}
```

Comment:
Since we have another pipeline group for `Checks`,pls make tihs name `{{ .Release.Name }}-rating` .